docs/user_guide/source/api/variable.rst

```diff
@@ -62,8 +62,8 @@ ADIOS2 is designed *out-of-the-box* for MPI applications. Thus different applica
 
       if( rank == 0 ) 
       {
-         adios2::Variable<unsigned int> varNodes = adios2::DefineVariable<unsigned int>("Nodes");
-         adios2::Variable<std::string> varFlag = adios2::DefineVariable<std::string>("Nodes flag");
+         adios2::Variable<unsigned int>& varNodes = adios2::DefineVariable<unsigned int>("Nodes");
+         adios2::Variable<std::string>& varFlag = adios2::DefineVariable<std::string>("Nodes flag");
```

Comment:
Please, check my original comment.....things are not changing for function arguments. This change was correct (no &) as those are eliminated for returned objects. Thanks!

Reply:
Then why was the last one  (ea70f8f) not accepted?

Reply:
I put a comment, not sure if you can see it under the Conversation tab.